app/javascript/mastodon/locales/en.json

```diff
@@ -125,6 +125,10 @@
   "community.column_settings.remote_only": "Remote only",
   "compose.language.change": "Change language",
   "compose.language.search": "Search languages...",
+  "compose_form.alt_text_warning.header": "Your attachments are missing a description!",
+  "compose_form.alt_text_warning.message": "Descriptions are for users with visual or audio impairments that prevent them from seeing or hearing an image, video, or audio file. It's recommended that you provide a description for all attached media files so everyone can enjoy using Mastodon.",
```

Comment:
Please update with more inclusive language (e.g. "users who are deaf, hard of hearing, blind or have low vision") per guidance in #20923. Words like "impairment" have negative connotations and are not best practice when talking about folks with disabilities. Thank you!

Reply:
@hs4man21 I've updated the phrasing, thanks for pointing this out 👍